package.json

```diff
@@ -26,9 +26,10 @@
     "@typescript-eslint/parser": "5.19.0",
     "eslint": "8.13.0",
     "eslint-config-prettier": "8.5.0",
-    "jest": "27.5.1",
+    "jest": "29.7.0",
+    "jest-environment-jsdom": "29.7.0",
```

Comment:
After `jest` v28, `jest-environment-jsdom` is no longer installed by default.  
Therefore, we need to install it separately.